build/lib/q2_gg2/__init__.py

```diff
@@ -0,0 +1,13 @@
+# ----------------------------------------------------------------------------
```

Comment:
It looks like files from the `build/` directory were accidentally included in the PR. Can the `build/` directory be removed from the pull request?